docs/standard/modernize-with-azure-and-containers/walkthroughs-technical-get-started-overview.md

```diff
@@ -276,80 +274,6 @@ With Kubernetes, developers can progress from thinking about physical and virtua
 
 Explore this content more in-depth on the GitHub wiki: <https://github.com/dotnet-architecture/eShopModernizing/wiki/04.-How-to-deploy-your-Windows-Containers-based-apps-into-Kubernetes-in-Azure-Container-Service-(Including-C-CD)>
 
-## Walkthrough 6: Deploy your Windows Containers-based apps to Azure Service Fabric
```

Comment:
Please re-look at some of the WIKI links in the Walkthroughs - they have broken links wherever referred. Not sure if this was broken after wiki changes in the repo. 
Specifically, check:
 - Walkthrough 3:  this link> https://github.com/dotnet/docs/pull/12670/files?file-filters%5B%5D=.md&hide-deleted-files=true&owned-by%5B%5D=#diff-76084fdb188f0fea5aff1cfc05dceec5  (2 referrers) 
-  Walkthrough 4: https://github.com/dotnet-architecture/eShopModernizing/wiki/05.-Deploying-the-Apps-to-ACI-(Azure-Container-Instances)TBD 
-  Walkthrough 5:  https://github.com/dotnet-architecture/eShopModernizing/wiki/04.-How-to-deploy-your-Windows-Containers-based-apps-into-Kubernetes-in-Azure-Container-Service-(Including-C-CD) (multiple referrers)

Since Service Fabric is removed, we should have a walkthrough mentioned for Deploying to Web Apps for Containers 
https://github.com/dotnet-architecture/eShopModernizing/wiki/Deploy-Windows-Container-to-Azure-App-Service